Populist Legal Power/common/laws/yMoG_PLP_welfare.txt

```diff
@@ -73,7 +74,7 @@ law_old_age_pension = {
 		state_working_adult_ratio_add = 0.005
 		state_dependent_political_participation_add = 0.02
 		state_food_security_add = 0.01
-		building_minimum_wage_mult = 0.1
+		building_minimum_wage_mult = 0.15
```

Comment:
Unless we can confirm that this is actually capped effectively and doesn't lead to the runaway issues you've mentioned before, I don't see a reason to follow vanilla with this particular change.

Reply:
```suggestion
		building_minimum_wage_mult = 0.08 # 0.15
```